kyuubi-common/src/main/scala/org/apache/kyuubi/package.scala

```diff
@@ -30,7 +30,8 @@ package object kyuubi {
     if (buildFileStream == null) {
       throw new KyuubiException(s"Can not load the core build file: $buildFile, if you meet " +
         s"this exception when running unit tests " +
-        s"please make sure you have run the `mvn package` or `mvn generate-resources` command.")
+        s"please make sure you have run the `mvn package` or " +
+        s"`mvn antrun:run@build-info -pl kyuubi-common` command.")
```

Comment:
```
Caused by: org.apache.kyuubi.KyuubiException: Can not load the core build file: kyuubi-version-info.properties, if you meet this exception when running unit tests please make sure you have run the `mvn package` or `mvn antrun:run@build-info -pl kyuubi-common` command.
	at org.apache.kyuubi.package$BuildInfo$.<init>(package.scala:31)
	at org.apache.kyuubi.package$BuildInfo$.<clinit>(package.scala)
```

Reply:
It seems that no matter what phase the `maven-antrun-plugin` is configured in, IntelliJ IDEA will not run when building the project, so the `kyuubi-version-info.properties` file will not be generated.

Reply:
If you specify the execution id to run ( `mvn antrun:run@build-info -pl kyuubi-common` ) , you need maven version 3.3.1 or higher.

> MNG-5768 specify execution-id for direct plugin goal invocation from command line
https://issues.apache.org/jira/browse/MNG-5768